src/components/List/src/ListItemSecondaryAction.tsx

```diff
@@ -2,6 +2,14 @@ import React from 'react';
 import MaterialListItemSecondaryAction from '@material-ui/core/ListItemSecondaryAction';
 import { BaseChildrenProps } from '@gemeente-denhaag/baseprops';
 
+import { listitemsecondaryaction_classes as classes } from './bem-mapping';
+
+import './mui-override.module.css';
+import './listitemsecondaryaction.module.css';
+import './listitem.module.css';
+import '@gemeente-denhaag/design-tokens-components';
```

Comment:
We don't need this import here. IIRC, all tokens are up to the implementing developer to import or define in their app.